packages/runtime/src/useCases/transport/relationshipTemplates/LoadPeerRelationshipTemplate.ts

```diff
@@ -67,11 +67,11 @@ export class LoadPeerRelationshipTemplateUseCase extends UseCase<LoadPeerRelatio
         }
 
         const content = token.cache.content;
-        return await this.loadTemplate(content.templateId, content.secretKey);
+        return await this.loadTemplate(content.templateId, content.secretKey, content.forIdentity);
     }
 
-    private async loadTemplate(id: CoreId, key: CryptoSecretKey) {
-        const template = await this.templateController.loadPeerRelationshipTemplate(id, key);
+    private async loadTemplate(id: CoreId, key: CryptoSecretKey, forIdentity?: CoreAddress) {
```

Comment:
I do not get this one - how could you load a peer template with a different identity? I guess loading a peer template would not need the forIdentity parameter.

Reply:
This is the personalization information from the template - similar to id and key

Reply:
@Magnus-Kuhn maybe it's beneficial to remove this private method and put the content inside the other method.